src/main/resources/application.yml

```diff
@@ -0,0 +1,16 @@
+spring:
+  datasource:
+    driver-class-name: org.h2.Driver
```

Comment:
Change this for a real database like MySQL or Postgres, you can use Docker to run the database.